framework/branch-key-store.md

```diff
@@ -84,20 +84,20 @@ If both [Storage](#storage) and [DynamoDb Client](#dynamodb-client) are configur
 If both [KeyManagement](#keymanagement) and [KMS Client](#kms-client) are configured initialization MUST fail.
 If both [KeyManagement](#keymanagement) and [Grant Tokens](#aws-kms-grant-tokens) are configured initialization MUST fail.
 
-If [Storage](#storage) is configured with an [EncryptedKeyStore](#encryptedkeystore)
-then this MUST be the configured [EncryptedKeyStore interface](./key-store/encrypted-key-store.md#interface).
+If [Storage](#storage) is configured with an [KeyStorageInterface](#KeyStorageInterface)
+then this MUST be the configured [KeyStorageInterface interface](./key-store/key-storage-interface.md#interface).
 
-If [Storage](#storage) is not configured with an [EncryptedKeyStore](#encryptedkeystore)
-a [default encrypted key store](./key-store/default-encrypted-key-store.md#initialization) MUST be created.
+If [Storage](#storage) is not configured with an [KeyStorageInterface](#KeyStorageInterface)
+a [default encrypted key store](./key-store/default-key-storage-interface.md#initialization) MUST be created.
```

Comment:
Question: Is the intention to leave this label as "default encrypted key store" or change it?
```suggestion
a [default key storage](./key-store/default-key-storage-interface.md#initialization) MUST be created.
```

Reply:
adjusting